migration-engine/json-rpc-api-build/methods/getDatabaseVersion.toml

```diff
@@ -3,7 +3,8 @@ description = "Get the database version for error reporting."
 requestShape = "getDatabaseVersionInput"
 responseShape = "getDatabaseVersionOutput"
 
-[recordShapes.getDatabaseVersionInput]
+[recordShapes.getDatabaseVersionInput.fields.datasource]
+shape = "DatasourceParam"
```

Comment:
`DatasourceParam` refers to [this definition](https://github.com/prisma/prisma-engines/blob/a7cdbff5715df9d90abccbf032d9172ae65aa848/migration-engine/json-rpc-api-build/methods/common.toml#L3-L17).